core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -598,6 +601,17 @@ public void endResponseWithLastChild() {
         propagateResponseSideLog(lastChild.partial());
     }
 
+    @Nullable
+    @Override
+    public TracingContext tracingContext() {
+        return this.tracingContext;
+    }
+
+    @Override
+    public void tracingContext(TracingContext tracingContext) {
```

Comment:
I'm uncertain about adding the `TracingContext` to the core module because different tracing libraries may have different fields and meanings. For instance, in Pinpoint, the traceId represents a collection of keys (as outlined here: https://d2.naver.com/helloworld/1194202).

Instead, what do you think about simply providing a utility class in the brave module to facilitate easy extraction of properties, rather than adding TracingContext to the core module? We can consider adding the API to the core module later, once it's well-stabilized.